elide-core/src/main/java/com/yahoo/elide/core/EntityDictionary.java

```diff
@@ -99,7 +72,17 @@ public EntityDictionary() {
      * @return binding class
      */
     public String getBinding(Class<?> entityClass) {
-        return bindEntityToJsonApi.get(lookupEntityClass(entityClass));
+        return entity(entityClass).jsonApi;
+    }
+
+    /**
+     * Get the Database Manager for an entity
+     *
+     * @param entityClass entity name
+     * @return List of relationship names for entity
+     */
+    public DatabaseManager getDatabaseManager(Class<?> entityClass) {
```

Comment:
Do we really want different DatabaseManagers per entity?  How would we manage transactions across them?  It seems like we would want an abstraction within a single manager.